docs/source/release_notes/release-notes.rst

```diff
@@ -63,11 +63,25 @@ develop
 
     *    - |userbreak| |fixed|
          - AtlasDB no longer embeds user-agents in metric names.
+           This affects both AtlasDB clients as well as TimeLock Server.
            All metrics are still available; however, metrics which previously included a user-agent component will no longer do so.
            For example, the timer ``com.palantir.timestamp.TimestampService.myUserAgent_version.getFreshTimestamp`` is now named ``com.palantir.timestamp.TimestampService.getFreshTimestamp``.
            This was necessary for compatibility with an internal log-ingestion tool.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2322>`__)
 
+    *    - |userbreak| |fixed|
+         - AtlasDB no longer embeds Cassandra host names in its metrics.
+           Aggregate metrics are retained in both CassandraClientPool and CassandraClientPoolingContainer.
+           This was necessary for compatibility with an internal log-ingestion tool.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/foo>`__)
+
+    *    - |userbreak| |fixed|
```

Comment:
The per-table metrics are still interesting, could be worth explaining that this information is still available in the logs and in the Sweep progress table.  How easy is it to read the Sweep tables?  Could we document it so we can easily see this information?

Reply:
Sweep Progress isn't too hard to read if you have cqlsh access, and the logs document everything. I'll add docs for reading sweep statistics.

Reply:
(That said, though, not all of the information may still be available, because Sweep can sweep the sweep priority table. The logs have everything, of course.)